jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/StatisticsHandler.java

```diff
@@ -295,13 +245,28 @@ public MinimumDataRateHandler(long minimumReadRate, long minimumWriteRate)
             _minimumWriteRate = minimumWriteRate;
         }
 
+        /**
+         * Creates a {@code MinimumDataRateHandler} with the specified read and write rates.
+         * @param minimumReadRate the minimum number of bytes to be read per second, or 0 for not checking the read rate.
+         * @param minimumWriteRate the minimum number of bytes to be written per second, or 0 for not checking the write rate.
+         * @param handler the handler to wrap.
+         */
+        public MinimumDataRateHandler(long minimumReadRate, long minimumWriteRate, Handler handler)
```

Comment:
Does this actually use any of the features of StatsHandler or EventHandler?  Could this not be just a stand alone handler now?

Reply:
It uses `getBytesRead()` and `getBytesWritten()` from `StatsHandler` so either we duplicate the bytes counting logic or we keep `MinimumDataRateHandler` as a subclass of `StatsHandler`.